<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>734</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FIR Filter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>921</width>
      <height>521</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>700</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="parambox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>921</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameters</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sampling Frequency (Hz):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter Length:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Shift Samples:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="samp_freq">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="fir_len">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;160&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="samp_shitf">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;80&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Low Frequency (Hz):</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="impulse_radio">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Impulse</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="step_radio">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>80</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Step</string>
     </property>
    </widget>
    <widget class="QPushButton" name="designFilter">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>40</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Design Filter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>High Frequency (Hz):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Window Function:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="low_freq">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;10&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="hi_freq">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Display:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="win_func_box">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>5</number>
     </property>
     <item>
      <property name="text">
       <string>Rectangular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Triangular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Welch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sine</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hann</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hamming</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blackman</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nuttall</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BlackmanNuttall</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BlackmanHarris</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FlatTop</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="display_box">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>100</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Both</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Both2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Both3</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="filtertypebox">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Filter Type</string>
     </property>
     <widget class="QRadioButton" name="lp_radio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>LP</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="hp_radio">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>HP</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="bp_radio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>BP</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="bs_radio">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>BS</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>parambox</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
